specification/schemas/Email.json

```diff
@@ -1,5 +1,5 @@
 {
   "type": "string",
-  "pattern": "^(([^<>()\\[\\]\\.,;:\\s@\"]+(\\.[^<>()\\[\\]\\.,;:\\s@\"]+)*)|(\".+\"))@((\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))$",
+  "pattern": "/^[a-zA-Z0-9.!#$%&'*+\\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/",
```

Comment:
I think the surrounding `/` characters need to be removed.
Also, regex101 gives me a pattern error (for both regular expressions).
Furthermore, the regex in the backoffice is too lenient, as it validates emails without an extension: `dont@mail`

Let's use the regex from https://html.spec.whatwg.org/multipage/input.html#valid-e-mail-address in both the spec and the backoffice. You can copy the custom ValidationRule in `src/backoffice.js`.

Reply:
```suggestion
  "pattern": "^[a-zA-Z0-9.!#$%&'*+\\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$",
```